connector/src/main/java/org/jboss/as/connector/services/workmanager/WildflyWorkWrapper.java

```diff
@@ -61,7 +63,7 @@ protected void runWork() throws WorkCompletedException {
                 try {
                     WildflyWorkWrapper.super.runWork();
                 } catch (WorkCompletedException e) {
-                    e.printStackTrace();
+                    ROOT_LOGGER.error(e.getLocalizedMessage());
```

Comment:
This should have a meaningful error message (i.e. a message added to the relevant logger), and should include the exception stack trace (like what you have done below).